fdbserver/SimKmsConnector.actor.cpp

```diff
@@ -123,13 +123,14 @@ ACTOR Future<Void> ekLookupByIds(Reference<SimKmsConnectorContext> ctx,
 	for (const auto& item : req.encryptKeyInfos) {
 		const auto& itr = ctx->simEncryptKeyStore.find(item.baseCipherId);
 		if (itr != ctx->simEncryptKeyStore.end()) {
+			ASSERT(item.domainId.present());
```

Comment:
If in the future we plan on making the domain id optional can we put a todo here?

Reply:
Good point, it is fall out from my PR.